scalding-hadoop-test/src/test/scala/com/twitter/scalding/reducer_estimation/ReducerEstimatorTest.scala

```diff
@@ -81,6 +81,43 @@ class SimpleGlobJob(args: Args, customConfig: Config) extends Job(args) {
     .write(counts)
 }
 
+class SimpleMemoryJob(args: Args, customConfig: Config) extends Job(args) {
+  import HipJob._
+
+  val inSrc = IterableSource(List(
+    "Direct trade American Apparel squid umami tote bag. Lo-fi XOXO gluten-free meh literally, typewriter readymade wolf salvia whatever drinking vinegar organic. Four loko literally bicycle rights drinking vinegar Cosby sweater hella stumptown. Dreamcatcher iPhone 90's organic chambray cardigan, wolf fixie gluten-free Brooklyn four loko. Mumblecore ennui twee, 8-bit food truck sustainable tote bag Williamsburg mixtape biodiesel. Semiotics Helvetica put a bird on it, roof party fashion axe organic post-ironic readymade Wes Anderson Pinterest keffiyeh. Craft beer meggings sartorial, butcher Marfa kitsch art party mustache Brooklyn vinyl.",
+    "Wolf flannel before they sold out vinyl, selfies four loko Bushwick Banksy Odd Future. Chillwave banh mi iPhone, Truffaut shabby chic craft beer keytar DIY. Scenester selvage deep v YOLO paleo blog photo booth fap. Sustainable wolf mixtape small batch skateboard, pop-up brunch asymmetrical seitan butcher Thundercats disrupt twee Etsy. You probably haven't heard of them freegan skateboard before they sold out, mlkshk pour-over Echo Park keytar retro farm-to-table. Tattooed sustainable beard, Helvetica Wes Anderson pickled vinyl yr pop-up Vice. Wolf bespoke lomo photo booth ethnic cliche."
+  ))
+
+  override def config = super.config ++ customConfig.toMap.toMap
+
+  TypedPipe.from(inSrc)
+    .flatMap(_.split("[^\\w]+"))
+    .map(_.toLowerCase -> 1)
+    .group
+    // force the number of reducers to two, to test with/without estimation
+    .withReducers(2)
+    .sum
+    .write(counts)
+}
+
+class SimpleFileNotFoundJob(args: Args, customConfig: Config) extends Job(args) {
```

Comment:
this isn't being used right? Should we add a test to include this scenario? Source is a Hfs source but the path points to a file that doesn't exist?